### ISSUE TITLE
CSM RHEL 8: ibm-csm-tools rpm dependency on numpy needs updating

### ISSUE THREAD

**Issue by besawn** [IBM/CAST#959]
```
# yum install /u/besawn/1.8.2-3514/ibm-csm-tools-1.8.2-3514.noarch.rpm 
Updating Subscription Management repositories.
Unable to read consumer identity
This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.
Last metadata expiration check: 0:40:13 ago on Tue 01 Sep 2020 01:34:57 PM EDT.
Error: 
 Problem: conflicting requests
  - nothing provides numpy needed by ibm-csm-tools-1.8.2-3514.noarch
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```

The `numpy` package has changed names between RHEL7 and RHEL8. We need to identify the correct package for RHEL8, update the rpm dependency to reflect the new package, and check to make sure the functions that are using `numpy` still function correctly.

This is a low severity problem as this package contains auxiliary tools that are not needed for production operation.

**Comment by besawn** [comment]
These are the files that are creating this dependency:
```
$ grep -r numpy csm*
csmd/setupRPM.cmake:set(CPACK_RPM_csm-tools_PACKAGE_REQUIRES "numpy")
csmutil/csm_log_utility/API_Statistics.py:import numpy as np
csmutil/csm_log_utility/Helper_Files/Aggregator_Stat.py:import numpy as np
csmutil/csm_log_utility/Helper_Files/Compute_Stat.py:import numpy as np
csmutil/csm_log_utility/Helper_Files/Master_Stat.py:import numpy as np
csmutil/csm_log_utility/Helper_Files/Utility_Stat.py:import numpy as np
```
Looks like the new package name on RHEL8 is `python2-numpy`:
```
# yum list python2-numpy
Updating Subscription Management repositories.
Unable to read consumer identity
This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.
Last metadata expiration check: 0:46:43 ago on Wed 07 Oct 2020 08:58:05 AM EDT.
Available Packages
python2-numpy.ppc64le                                      1:1.14.2-13.module+el8.1.0+3323+7ac3e00f                                      xCAT-rhels8.2-path0--install-shared_repo-ISOs-rhels8.2.0-ppc64le-AppStream
python2-numpy.ppc64le                                      1:1.14.2-13.module+el8.1.0+3323+7ac3e00f                                      xCAT-rhels8.2-path1       
```

I think you can change this so that it finds the right version of numpy for python2 on both RHEL 7 and RHEL 8 by making the following change to csmd/setupRPM.cmake:
```
# ibm-csm-tools rpm settings
set(CPACK_RPM_csm-tools_PACKAGE_ARCHITECTURE "noarch")
if(${EXPLICIT_PYTHON_VERSION})
    set(CPACK_RPM_csm-tools_PACKAGE_REQUIRES "python2-numpy")
else()
    set(CPACK_RPM_csm-tools_PACKAGE_REQUIRES "numpy")
endif()
```

**Comment by besawn** [comment]
This was fixed via PR #980.
